projects/igniteui-angular/src/lib/directives/dragdrop/dragdrop.directive.ts

```diff
@@ -62,60 +62,181 @@ export class IgxDropEventArgs {
 })
 export class IgxDragDirective implements OnInit, OnDestroy {
 
+    /**
+     * @hidden
+     */
     @Input('igxDrag')
     public data: any;
 
+    /**
+     * An @Input property that indicates when the drag should start
+     * By default the drag starts after the draggable element is moved by 5px
+     * ```html
+     * <div igxDrag [dragTolerance]="100">
+     *         <span>Drag Me!</span>
+     * </div>
+     * ```
+     */
     @Input()
     public dragTolerance = 5;
 
+    /**
+     * @hidden
+     */
     @Input()
     public ghostImageClass = '';
```

Comment:
This should also not be hidden. This sets a custom class that will be added to the dragGhost element.